### PR TITLE
Adding TEFCA Viewer unittest for formatName

### LINKED ISSUE #1990: Write unit tests for formatName
Add tests for `formatName` in `format-service.tsx`.

### PR DESCRIPTION
**Pull request by robertandremitchell**
# PULL REQUEST

## Summary
Adds additional unittest to tefca-viewer for formatName.

## Related Issue
Fixes #1990

## Additional Information
I added a bit more to the initial function to handle missingness.

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/tefca-viewer/src/app/format-service.tsx [hunk 1]
```diff
@@ -54,7 +54,9 @@ export function formatCodeableConcept(concept: CodeableConcept | undefined) {
 export function formatName(names: HumanName[]): string {
   let name = "";
   if (names.length > 0) {
-    name = names[0].given?.join(" ") + " " + names[0].family;
+    const givenNames = names[0].given?.filter((n) => n).join(" ") ?? "";
```
Comment (DanPaseltiner) [inline containers/tefca-viewer/src/app/format-service.tsx:57]: What does using `filter` do for us here?

File: containers/tefca-viewer/src/app/tests/format-service.test.tsx
Comment (DanPaseltiner) [inline containers/tefca-viewer/src/app/tests/format-service.test.tsx:1]: 🚀

**Comment by DanPaseltiner** [comment]
Looking at the tests that have run I think we need to add unit tests into the CI workflow for the TEFCA Viewer

File: containers/tefca-viewer/src/app/format-service.tsx (hunk 1 above)
Comment (robertandremitchell) [reply containers/tefca-viewer/src/app/format-service.tsx:57]: it's probably a very very edge case, but when I was looking up ways to gracefully handle null/undefined data, I came across that filter as way to handle a case where a given name might be [John, `undefined`].

**Review by DanPaseltiner — APPROVED** [review]

**Pull request merged**